exasol/analytics/schema/statements.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import annotations
+
+from typing import Any
+
+from exasol.analytics.schema.column import Column
+from exasol.analytics.schema.column_name import ColumnName
+from exasol.analytics.schema.values import quote_value
+
+
+class InsertStatement:
+    """
+    Columns and values for creating an SQL INSERT statement.
+    """
+
+    def __init__(self, columns: list[Column], separator: str = ",\n  "):
```

Comment:
I wouldn't bother about the separator or set the default value to ', ' or ','. By setting the default to the new line you insist on some kind of formatting which may not actually look nice. And, of course, technically it's not needed.

Reply:
See next push.